docs/marketplace/dev/feature-integration-guides/202304.0/marketplace-product-offer-prices-feature-integration.md

```diff
@@ -0,0 +1,1185 @@
+
+```xml
+<?xml version="1.0"?>
+<database xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          name="zed"
+          xsi:noNamespaceSchemaLocation="http://static.spryker.com/schema-01.xsd"
+          namespace="Orm\Zed\PriceProductOffer\Persistence"
+          package="src.Orm.Zed.PriceProductOffer.Persistence"
+>
+  <table name="spy_price_product_offer" class="SpyPriceProductOffer">
+    <behavior name="event">
+      <parameter name="spy_price_product_offer_all" column="*"/>
+    </behavior>
+  </table>
+</database>
+```
+
+Apply database changes and to generate entity and transfer changes:
+
+```bash
+console transfer:generate
+console propel:install
+console transfer:generate
+```
+
+{% info_block warningBox "Verification" %}
+
+Verify that the following changes have been implemented by checking your database:
+
+| DATABASE ENTITY | TYPE | EVENT |
+|-|-|-|
+| spy_price_product_offer | table | created |
+| spy_product_concrete_product_offer_price_storage | table | created |
+
+Make sure that the following changes were applied in transfer objects:
+
+| TRANSFER | TYPE | EVENT | PATH |
+|-|-|-|-|
+| PriceProductOffer | class | created | src/Generated/Shared/Transfer/PriceProductOfferTransfer |
+| PriceProductOfferCriteria | class | created | src/Generated/Shared/Transfer/PriceProductOfferCriteriaTransfer |
+| PriceProductOfferCollection | class | created |  src/Generated/Shared/Transfer/PriceProductOfferCollectionTransfer |
+| PriceProductStoreCriteria | class | created | src/Generated/Shared/Transfer/PriceProductStoreCriteriaTransfer |
+| Pagination | class | created | src/Generated/Shared/Transfer/PaginationTransfer |
+| PriceProductCriteria.productOfferReference | property | created | src/Generated/Shared/Transfer/PriceProductCriteriaTransfer |
+| PriceProduct.concreteSku | property | created | src/Generated/Shared/Transfer/PriceProductTransfer |
+| PriceProductDimension.productOfferReference | property | created | src/Generated/Shared/Transfer/PriceProductDimensionTransfer |
+| PriceProductDimension.idProductOffer | property | created | src/Generated/Shared/Transfer/PriceProductDimensionTransfer |
+| PriceProductDimension.idPriceProductOffer | property | created | src/Generated/Shared/Transfer/PriceProductDimensionTransfer |
+| ProductOffer.prices | property | created | src/Generated/Shared/Transfer/ProductOfferTransfer |
+| PriceProductFilterIdentifier.productOfferReference | property | created | src/Generated/Shared/Transfer/PriceProductFilterIdentifierTransfer |
+| ProductOfferStorage.price | property | created | src/Generated/Shared/Transfer/ProductOfferStorageTransfer |
+| PriceProductFilter.productOfferReference | property | created |src/Generated/Shared/Transfer/PriceProductFilterTransfer |
+
+{% endinfo_block %}
+
+### 3) Add Zed translations
+
+Generate a new translation cache for Zed:
+
+```bash
+console translator:generate-cache
+```
+
+### 4) Configure export to Redis
+
+Set up event listeners
+
```

Comment:
Add an introductory phrase. Other IGs have the following intro phrase for this step. Maybe it fits here as well:
```suggestion
{% info_block infoBox %}

In this step, you enable publishing of table changes—create, edit, delete to `spy_category_image_storage` and synchronization of data to Storage.


{% endinfo_block %}
```